cmd/manager/main.go

```diff
@@ -322,7 +322,7 @@ func authFilePathIfPresent(logger logr.Logger) string {
 		return ""
 	}
 	if err != nil {
-		logger.Error(err, "could not stat auth file path", "path", authFilePath)
+		logger.Error(err, "Errors accessing the file in", "path", authFilePath)
```

Comment:
We should follow the format of other errors that are written via this logger (and have a separate convo, if you like, about the format we use)

```suggestion
		logger.Error(err, "unable to access the global auth file", "path", authFilePath)
```